File: src/models/Message/MessagePosting.ts
interface IMessagePosting {
    title: string;
    body: string;
    recipient: string;
}

export class MessagePosting implements IMessagePosting {
    title: string;
    body: string;
    recipient: string;

    constructor(title: string, body: string, recipient: string) {
        this.title = title;
        this.body = body;
        this.recipient = recipient;
    }
}

File: src/main/java/com/schoolproject/airbnbclone/dtos/message/request/MessagePosting.java
package com.schoolproject.airbnbclone.dtos.message.request;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.io.Serializable;

@Data
@AllArgsConstructor
public class MessagePosting implements Serializable {

    private final String title;
    private final String body;
    private final String recipient;

}
